app/views/admin/administrators/administrators/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: admin_administrators_administrators_path)
+%>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Create a new administrator user</h1>
+    <p class="govuk-body">This should be someone from the Department for Education</p>
+    <%= form_for @user, url: admin_administrators_administrators_path, method: :post do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.govuk_text_field(
```

Comment:
This could be on one line

Reply:
I think this is easier to read than `<%= f.govuk_text_field(:full_name, label: { text: "Full name" }) %>`. What do you think?

Reply:
Hmm if the amount of variables are small - it's easier to scan one line - not a hard comment - simply a suggestion